cmd/vc-rest/startcmd/start.go

```diff
@@ -501,7 +502,7 @@ func supportedMode(mode string) bool {
 // acceptsDID returns if given did method is accepted by VC REST api
 func acceptsDID(method string) bool {
 	return method == didMethodVeres || method == didMethodElement || method == didMethodSov ||
-		method == didMethodWeb || method == didMethodKey
+		method == didMethodWeb || method == didMethodKey || method == didMethodFactom
```

Comment:
I'm somewhat curious why we don't just reply true :)

Reply:
do we have multiple DID acceptors in this service?
if we need to do it this way, we need a followup to make this configurable.

Reply:
May be original idea was to have some whitelist. Probably we can move this to a config.

Reply:
https://github.com/trustbloc/edge-service/issues/406